public-lighting-demo-app/web-demo-app/src/main/java/org/opensmartgridplatform/webdemoapp/application/services/OsgpAdminClientSoapService.java

```diff
@@ -13,24 +13,20 @@
 
 import org.opensmartgridplatform.adapter.ws.schema.admin.devicemanagement.UpdateKeyRequest;
 
-import ma.glasnost.orika.MapperFacade;
-
 /**
  * Service class used for sending and receiving SOAP messages to/from the
  * platform.
  *
  */
 public class OsgpAdminClientSoapService {
+	
+    private final String PUBLIC_KEY =
+        "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEFhUImXFJdqmputquVAc2CPdnn9Ju00M3m/Ice7wABNN+oAYKQbw/OceqvZmFF1+r4nO/vCm/f1JO5nEorE2jNQ==";
+    private final int OSLP_ELSTER_PROTOCOL = 4;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.4.24.184/sonarqube/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "OSLP_ELSTER_PROTOCOL" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.4.24.184/sonarqube/coding_rules#rule_key=squid%3AS00116)